blueocean-pipeline-api-impl/src/test/resources/io/jenkins/blueocean/rest/impl/pipeline/restartStage.jenkinsfile

```diff
@@ -41,7 +41,7 @@ pipeline {
     }
     stage('Deploy') {
       parallel {
-        stage('Deploy') {
+        stage('DeployX') {
```

Comment:
What's this change about?

Reply:
The new unit tests are based on stage names for readability, so they won't work on pipelines that have multiple stages with the same name.